bedrock/mozorg/templates/mozorg/mpl/headers/index.html

```diff
@@ -1,6 +1,6 @@
 {# This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+# file, You can obtain one at https://mozilla.org/MPL/2.0/. -#}
```

Comment:
In fact we could update these to the newest full URL: https://www.mozilla.org/en-US/MPL/2.0/

Reply:
well... we can leave out the `en-US/` bit as it will redirect, but we could add the www. Thoughts?

Reply:
Maybe it should match the blurb in the license though?

Reply:
Good point. Seems fine 👍